glide.lock

```diff
@@ -41,7 +41,7 @@ imports:
   - httputil
```

Comment:
if we have a new direct dependency (like `tomb.v2` in this case) we add it to glide.yaml and then do a glide up, because glide.lock gets removed and rebuilt from glide.yaml during build process usually (not sure if we do that with this repo yet)